px.py

```diff
@@ -251,9 +251,7 @@ def dprint(*objs):
         try:
             print(multiprocessing.current_process().name + ": " +
                   threading.current_thread().name + ": " + str(int(time.time())) +
-                  ": " + sys._getframe(1).f_code.co_name + ": ", end="")
-            print(*objs)
-            sys.stdout.flush()
+                  ": " + sys._getframe(1).f_code.co_name + ": ", *objs, flush=True)
```

Comment:
Crashes on Python 2.7 since `flush=True` isn't supported. Have to revert back to using `sys.stdout.flush()`

Reply:
Done.